UKHomeOffice/nmsw-ui
# NMSW-1344 Remove completed page & just go to task list

# Ticket



----

## AC

**Scenario: refresh from holding page**

Given I have uploaded a FAL5
And there are no errors
When I click the continue button
And there are checkins pending
Then I am shown the holding page
When I click the 'refresh' button 
And there are no checking pending and no AMG errors
Then I am taken to the task list page
And the crew status is complete

**Scenario: continue from upload (no checkins pending)**

Given I have uploaded a FAL5
And there are no errors
When I click the continue button
And there are no checking pending and no AMG errors
Then I am taken to the task list page
And the crew status is complete

----

## To test

- create a declaration
- upload a small FAL5
- click continue
- > you should be taken to the task list and it should show as complete (assuming it's not got any check ins pending)
- click on FAL5 link on task list
- > you should be taken to FAL5 upload page
- upload a large FAL5
- click continue
- > you should be taken to the holding page
- wait for AMG checking to be complete (0 checkins, 0 error responses)
- click refresh
- > you should be taken to the task list page and it should show as complete

----

## Notes
